src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/ConvertFromJsonCommand.cs

```diff
@@ -71,35 +71,47 @@ protected override void EndProcessing()
             // 2. The complete input is a collection which represents a single Json content. This is typically the majority of the case.
             if (_inputObjectBuffer.Count > 0)
             {
+                ErrorRecord error = null;
+                ArgumentException exception = null;
+
                 if (_inputObjectBuffer.Count == 1)
                 {
-                    ConvertFromJsonHelper(_inputObjectBuffer[0]);
+                    ConvertFromJsonHelper(_inputObjectBuffer[0], out error);
```

Comment:
The 'out error' variable is not needed because if there is an error in 'ConvertFromJsonHelper' it is thrown.  Either return the exception in the error variable or throw it, but not both.